impress-org/givewp
# SelectInput styles are improved

Resolves #4976 

## Description
This PR introduces minor improvements to the SelectInput component styles, and adds a `label` prop to the SelectInput component for easily attaching labels to inputs (as seen in the Location step of the Onboarding Wizard). 

## Affects
This PR affects frontend logic of the Onboarding Wizard, specifically how SelectInput components are displayed, and how their markup is rendered. As a result, steps which utilize the SelectInput component have also been updated to respond to these changes.

## Visuals
<img width="663" alt="Screen Shot 2020-08-05 at 2 16 05 PM" src="https://user-images.githubusercontent.com/5186078/89465233-9c588f00-d726-11ea-8903-2018a964f791.png">

## Pre-review Checklist

## Testing Instructions
Navigate to the Onboarding Wizard `wp-admin/?page=give-onboarding-wizard`, and review each step of the wizard which uses the SelectInput component. Do these inputs more closely reflect select input design put forward in the Figma boards (see: https://www.figma.com/file/4kIK1E96lxEuMKad2msa0a/Epic%3A-Plugin-Onboarding?node-id=397%3A2094)?
